app/gui/src/documentation_debug_scene.rs

```diff
@@ -0,0 +1,213 @@
+//! Example scene showing a documentation panel of the component browser.
+
+use ensogl::prelude::*;
+use wasm_bindgen::prelude::*;
+
+use crate::doc_section;
+use crate::mock_suggestion_database;
+use crate::model::suggestion_database;
+use crate::model::suggestion_database::Entry;
+use crate::model::SuggestionDatabase;
+use ensogl::application::Application;
+use ensogl::data::color;
+use ensogl::display;
+use ensogl::display::navigation::navigator::Navigator;
+use ensogl::display::object::ObjectOps;
+use ensogl::display::shape::StyleWatchFrp;
+use ensogl::display::shape::*;
+use ensogl::frp;
+use ensogl::shape;
+use ensogl::system::web;
+use ensogl_hardcoded_theme::application::component_browser as theme;
+use ide_view::documentation;
+use ide_view::graph_editor::component::visualization::Registry;
+use std::f32::consts::PI;
+
+
+
+// =======================
+// === DatabaseWrapper ===
+// =======================
+
+/// A wrapper around the suggestion database that allows iterate over entries and display their
```

Comment:
1. "wrapper for" is better than "wrapper around"
2. "allows iteration" or "allows to iterate"

Were the sentences checked with gramarly?

Reply:
Fixed